parm/config/config.resources.emc.dyn

```diff
@@ -160,7 +160,7 @@ elif [ $step = "analcalc" ]; then
 elif [ $step = "analdiag" ]; then
 
     export wtime_analdiag="00:10:00"
-    export npe_analdiag=72
+    export npe_analdiag=115		# Should be at least twice npe_ediag
```

Comment:
Please set npe_analdiag=96 so that it is twice npe_ediag.  Same comment applies to `config.resources.nco.static`.

Reply:
Will change this back to 96. Thanks @RussTreadon-NOAA !